trade-finance-manager-ui/server/constants/utilisation-report-display-frequency.ts

```diff
@@ -0,0 +1,4 @@
+export const UTILISATION_REPORT_DISPLAY_FREQUENCY = {
+  QUARTERLY: 'Quarterly',
+  MONTHLY: 'Monthly',
+} as const;
```

Comment:
Not a blocker - we should create DRY shared `FEE_FREQUENCY` constants, just searching the codebase I can see 5x different instances of `QUARTERLY: 'Quarterly'`.

`FEE_FREQUENCY` in ACBS functions seems to be the most comprehensive, I think this should be extracted into shared constants, not sure if we have such a shared package in place yet @abhi-markan

Reply:
Interesting I had not seen fee frequency as a thing. I think here we do want a separate constant however because this is a report frequency not a fee frequency so only overlaps by chance rather than intent and they have separate business meanings I imagine as they allow different ranges of frequency. Generally not a good idea to link things that only relate coincidentally as it makes it harder to maintain down the line, even though it does look more DRY at a glance.

Reply:
@RLavender98 I was thinking we could have generic `FREQUENCY` constants, as regardless of where such frequencies are used, they are all simply frequencies of a year.

E.g from ACBS (just renamed the const here):

```ts
const FREQUENCY = {
  WEEKLY: 'Weekly',
  BI_WEEKLY: 'Bi-Weekly',
  MONTHLY: 'Monthly',
  BI_MONTHLY: 'Bi-Monthly',
  QUARTERLY: 'Quarterly',
  SEMI_ANNUALLY: 'Semi-annually',
  ANNUALLY: 'Annually',
  BI_ANNUALLY: 'Bi-annually',
};
```

Such frequencies should be aligned throughout all products, e.g it's not wise to have 1x service saying "Bi weekly", but another says "Bi-weekly", it should be standardised